iterative/resource_task.go

```diff
@@ -55,7 +55,7 @@ func resourceTask() *schema.Resource {
 				Type:     schema.TypeInt,
 				ForceNew: true,
 				Optional: true,
-				Default:  30,
+				Default:  -1,
```

Comment:
should this be reverted?

Reply:
No, -1 means automatic

Reply:
would be nice to update the docs too then

- docs/guides/getting-started.md
- docs/resources/task.md
- README.md

all still mention `30` GB and no mention of `-1`

Reply:
Yes

Reply:
just pushed a couple of commits - please check

Reply:
Solved with https://github.com/iterative/terraform-provider-iterative/pull/533/commits/5b96c7e232c4b5c217f8131cc3a4585bfdf67d6c